cli/cli.go

```diff
@@ -285,6 +285,21 @@ func NewApp(version, environment, hash, salt string,
 				Action:       cmd.SetLANDiscovery,
 				BashComplete: cmd.SetBoolAutocomplete,
 			},
+			{
+				Name:         "postquantum",
+				Aliases:      []string{"pq"},
+				Usage:        SetPostquantumVpnUsageText,
+				Action:       cmd.SetPostquantumVpn,
+				BashComplete: cmd.SetBoolAutocomplete,
+				ArgsUsage:    MsgSetBoolArgsUsage,
+				Description: fmt.Sprintf(
+					MsgSetBoolDescription,
+					SetProtocolUsageText,
```

Comment:
More suitable text should be used, now it is reused from other setting.

Reply:
Done